lib/shared/bucketing-assembly-script/assembly/types/dvcUser.ts

```diff
@@ -97,6 +97,7 @@ export class DVCPopulatedUser extends JSON.Value implements DVCUserInterface {
     platformVersion: string
     sdkType: string
     sdkVersion: string
+    hostname: string | null
```

Comment:
when'd it be null?

Reply:
in the case where `os.hostname` returns a null, since we can't guarantee it will give a value as opposed to a hard coded value